From f870e868c2f37dcdbce81eb47ad4e9ed9a0d21df Mon Sep 17 00:00:00 2001
From: Saleem Abdulrasool <compnerd@compnerd.org>
Date: Wed, 13 Jan 2010 20:14:18 -0800
Subject: [PATCH] explicitly initialise bonobo

If bonobo is not initialised, then gnome-panel will fail to load any applets.
It will report that it was unable to load the applet because it failed to
initialise the ActivationContext.
---
 gnome-panel/main.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/gnome-panel/main.c b/gnome-panel/main.c
index 3b1e4d3..8d7f750 100644
--- a/gnome-panel/main.c
+++ b/gnome-panel/main.c
@@ -77,13 +77,10 @@ main (int argc, char **argv)
 	g_option_context_add_main_entries (context, options, GETTEXT_PACKAGE);
 
 	gtk_init (&argc, &argv);
-#if 0
-	/* Hrm, not needed? */
 	if (!bonobo_init (&argc, argv)) {
 		g_printerr ("Cannot initialize bonobo.\n");
 		return 1;
 	}
-#endif
 
 	error = NULL;
 	if (!g_option_context_parse (context, &argc, &argv, &error)) {
-- 
1.6.6

